test/fixtures/3.0/ping/.gitignore

```diff
@@ -0,0 +1,64 @@
+# Logs
```

Comment:
1. Please use a more descriptive directory name than `test/fixtures/3.0/ping`. We already have files like ` test/fixtures/3.0/ping.json` and `test/fixtures/3.0/ping.yaml`, it's not clear to me at all what to expect inside ` test/fixtures/3.0/ping` directory.

    Proposed name: `lb4-ping-app`

2. Do we really need all these `.gitignore` entries? Can you please pick only those that are necessary and move them to the top-level `.gitignore` file please?